src/interfaces.ts

```diff
@@ -27,6 +27,15 @@ export interface RetractRequest {
   secondaryId?: string; //It will be deprecated soon
   subNotificationId?: string;
 }
+export interface SubNotificationCreateInput {
```

Comment:
Following the other interfaces in this file, you can name this CreateSubNotificationRequest.